### ISSUE TITLE
[Feature request] Pop up a notification when there is no GSM signal

### ISSUE THREAD

**Issue by Siltaar** [scheich/Throughput#13]
Hi,

Throughput monitors the data of connections, and it's notification disappears when there is no more data connection.

What about adding another notification when there is no more data connection, or even no more signal at all ? (WiFi or GSM/LTE) ?

The use case is having a notification led blinking when the phone can't receives data (and even calls, which is my need), in order to quickly see if the phone became useless.

It may be because we're somewhere without signal, but in fact, in my house I don't have signal everywhere, every time and should pay attention to it.

I notice that LineageOS / Android 8.1 allows to display an icon with the throughput, but there's more details in the Throughput application, and it would be one more interest in the app to be able to monitor also the absence of connectivity. Android can vibrate when GSM signal comes or go away, but no blinking led during no signal periods.

**Comment by scheich** [comment]
Sounds good and makeable:

1. Show notification similar to airplane mode, if theres no signal.
2. Play sound/vib/led if you get or miss signal.

Correct?

**Comment by Siltaar** [comment]
It is. (I plan to use the led)

Android 8.1 allows to define colors and blinking durations of notification led for every app, one by app, it was not the case with Android 7 and previous.
So it would be good to have a way to set a notification only when there is no signal, and no notification if the signal comes back (but the removal of any Throughput app related notifications).

**Comment by scheich** [comment]
Forgot, you want get an notification, if screen is off. 

If screen is off, the app pause activity to avoid too much battery usage.

But that is what you want, isnt it?

**Comment by Siltaar** [comment]
Yes it is. The idea is that the phone tells, via its notification LED (when the screen is off), that it is out of order because it is not registered on a GSM network.
